iceberg/bodo_iceberg_connector/py4j_support.py

```diff
@@ -102,7 +102,7 @@ def launch_jvm() -> JavaGateway:
         gateway_port = pt.cast(
             int,
             launch_gateway(
-                jarpath=full_path,
+                classpath=full_path,
```

Comment:
This fixed an issue for me locally (not sure why it doesn't happen on CI or BodoSQL side). `jarpath` is for the py4j jar itself:
https://www.py4j.org/py4j_java_gateway.html#py4j.java_gateway.launch_gateway